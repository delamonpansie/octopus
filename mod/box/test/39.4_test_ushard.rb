#!/usr/bin/ruby
# coding: utf-8

$: << File.dirname($0)
require '39_test_ushard'

$one_env.meta 'shard 1 create por'

# create existing shard
log_try { $two_env.meta 'shard 1 create por' }


