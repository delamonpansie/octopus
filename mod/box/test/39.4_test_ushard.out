# one.ping()
:pong

# two.ping()
:pong

# one.meta(shard 1 create por)
ok

# two.meta(shard 1 create por)
error: 0x202 shard already exists

